unattended-upgrade

```diff
@@ -388,13 +391,29 @@ def should_stop():
     Return True if u-u needs to stop due to signal received or due to the
     system started to run on battery.
     """
+    global SIGNAL_STOP_REQUEST
+    global MISSING_NMCLI_PRINTED
     if SIGNAL_STOP_REQUEST:
         logging.warning("SIGNAL received, stopping")
         return True
     if apt_pkg.config.find_b("Unattended-Upgrade::OnlyOnACPower", True) and \
        subprocess.call("on_ac_power") == 1:
         logging.warning("System is on battery power, stopping")
         return True
+    if apt_pkg.config.find_b(
+            "Unattended-Upgrade::Skip-Updates-On-Metered-Connections", True):
```

Comment:
@julian-klode apt could use this config option to prevent downloads for u-u, too.